weaver/src/main/java/org/aspectj/weaver/tools/cache/SimpleCache.java

```diff
@@ -120,10 +120,7 @@ private static class StoreableCachingMap extends HashMap {
 		private static int DEF_STORING_TIMER = 60000; //ms
 		private int storingTimer;
 
-		private transient Trace trace;
-		private void initTrace(){
-			trace = TraceFactory.getTraceFactory().getTrace(StoreableCachingMap.class);
-		}
+		private static final Trace trace = TraceFactory.getTraceFactory().getTrace(StoreableCachingMap.class);
```

Comment:
I am fine with the other changes. Here, I am not sure if @aclement had anything special in mind by choosing a transient instance field rather than a static one. Andy, I know 2012 was long ago. Can you remember anything in the context of [Bugzilla 386341](https://bugs.eclipse.org/bugs/show_bug.cgi?id=386341)? If you are OK with this particular change here, I would merge it.

Reply:
Would be ok for merge if I revert this one?

Reply:
@turbanoff, I am reaching out to tell you that I am not ignoring your PRs because I do not like them or so. I simply would prefer Andy to review and merge them. But currently being the only active maintainer and busy with my normal job as well, I simply prioritise scratching my own itches, fixing bugs or working on little improvements of my personal interest. I am sure your improvements are valuable, but assessing whether they might have undesired side effects in parts of the code I do not know is simply time I currently choose to spend elsewhere, be it on AspectJ or privately. But your work is not lost. Please do not feel discouraged to open PRs. This is about my own situation, not about the quality of your contributions. Please accept my apologies. 🙂